dags/bqetl_fivetran_costs.py

```diff
@@ -48,6 +48,19 @@
     doc_md=docs,
     tags=tags,
 ) as dag:
+
+    fivetran_costs_derived__daily_active_rows__v1 = bigquery_etl_query(
```

Comment:
Or if you want to manually fix the DAG CI error, this should do it.
```suggestion
) as dag:
    fivetran_costs_derived__daily_active_rows__v1 = bigquery_etl_query(
```